lang/cs/Org.Apache.REEF.IMRU.Tests/TestTaskStates.cs

```diff
@@ -0,0 +1,251 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Org.Apache.REEF.IMRU.OnREEF.Driver.StateMachine;
+using Xunit;
+
+namespace Org.Apache.REEF.IMRU.Tests
+{
+    /// <summary>
+    /// The test cases in this classes test TaskState and transitions
+    /// </summary>
+    public class TestTaskStates
+    {
+        /// <summary>
+        /// This is to test a successful life cycle of task state transitions.
+        /// For TaskNew->TaskSubmitted->TaskRunning->TaskCompleted
+        /// </summary>
+        [Fact]
+        public void TestNewToCompleteScenario()
+        {
+            var taskState = new DriverTaskState();
+            Assert.True(taskState.CurrentState.Equals(TaskState.TaskNew));
```

Comment:
Please add message to `Assert.True`. Something like "State of newly created task should be TaskNew"